src/components/Tree/Tree.cy.tsx

```diff
@@ -7,11 +7,12 @@ import { Tree, TreeFlatListItem } from './Tree';
 
 type ComponentProps = {
     nodes: DraggableItem<TreeFlatListItem>[];
+    activeNodeIds?: NullableString[];
     onDrop?: (sourceItemId: string, parentId: NullableString, positionBeforeId: NullableString) => void;
     onEditableSave?: (itemId: string, value: string) => void;
 };
-const Component: FC<ComponentProps> = ({ nodes, onDrop, onEditableSave }) => {
-    const [selectedIds, setSelectedIds] = useState<NullableString[]>([]);
+const Component: FC<ComponentProps> = ({ nodes, onDrop, onEditableSave, activeNodeIds }) => {
```

Comment:
That would be great to emove `FC` from here too.

Reply:
Done :-)